_includes/sidebar.html

```diff
@@ -9,10 +9,9 @@
   <div class="more">
     <a href="/get">
       Sandstorm is a self-hostable web productivity suite.
-      <span class="signup" href="https://oasis.sandstorm.io">Sign up</span>
-      <span class="selfhost" href="/install">or host your own</span>
+      <span class="selfhost" href="/install">Host your own</span>
```

Comment:
I am tempted to suggest you abuse class=signup here. If you look at https://sandstorm.io/news/ you'll notice that the signup gets a nice purple button, and selfhost is just a web link. I know which one I think will look better there.

Reply:
Or even better, I could just change the name and not be abusive about it. It looks like nothing else uses these styles anyway. Done.